readme.md

```diff
@@ -181,11 +127,3 @@ There are only a few lines of basic code, which you need to either comment out o
 1. Run `npm uninstall koa @koa/router @discordx/koa @types/koa @types/koa__router`
```

Comment:
i didnt full review the code so I might be commenting something wrong, but perhaps we can remove a bunch of dependencies if we are not using rest api server.

also let's make a proper readme file 👍🏻